cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/azure-mc-basic/AzureBasicReview.config.tsx

```diff
@@ -0,0 +1,53 @@
+import { CommonValueDisplayFunctions, ConfigGroup } from '../../../../shared/components/ConfigReview/ConfigGrid';
+import { ConfigDisplayData } from '../../../../shared/components/ConfigReview/ConfigDisplay';
+import { OsImage } from '../../../../shared/components/FormInputComponents/OsImageSelect/OsImageSelect';
+import { AZURE_FIELDS } from './AzureManagementClusterBasic.constants';
+
+const configGroupCredentials: ConfigGroup = {
+    label: 'Azure Credentials',
+    pairs: [
+        { label: 'Tenant Id', field: AZURE_FIELDS.TENANT_ID },
+        { label: 'Client Id', field: AZURE_FIELDS.CLIENT_ID },
+        { label: 'Client Secret', field: AZURE_FIELDS.CLIENT_SECRET },
+        { label: 'Subscription', field: AZURE_FIELDS.SUBSCRIPTION_ID },
+        { label: 'Environment', field: AZURE_FIELDS.AZURE_ENVIRONMENT },
+        { label: 'Region', field: AZURE_FIELDS.REGION },
+        { label: 'Public SSH Key', field: AZURE_FIELDS.SSH_PUBLIC_KEY, createValueDisplay: CommonValueDisplayFunctions.TRUNCATE(24) },
+    ],
+};
+
+const configGroupClusterSettings: ConfigGroup = {
+    label: 'Cluster settings',
+    pairs: [
+        { label: 'Name', field: AZURE_FIELDS.CLUSTER_NAME },
+        { label: 'Node Type', field: AZURE_FIELDS.NODE_PROFILE },
+        {
+            label: 'Image (AMI)',
```

Comment:
For this label I think we should go with OS Image. AMI is an AWS-specific term. Azure doesn't seem to have a clear term and vsphere OS image could be vSphere OS Image Template if we wanted to be cute. I think simply OS Image is the safe term to use across providers. I should have thought through this on the OS image component label usage in previous PRs.